k8s/namespaces/rd/rd-location-ref-data-load/demo-00.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "0/10 * * * *"
+      schedule: "0/11 * * * *"
       environment:
-        DELETE_ORG: false
\ No newline at end of file
+        DELETE_ORG: false
```

Comment:
you can just change this to true to restart the pod instead of changing the job :)

Reply:
Sure,Thanks @SujeethKumarAri for the information.